app/memory_order.py

```diff
@@ -13,12 +13,6 @@ def order(self):
         return self._order
 
     def _str_to_endian(self, order_string: str):
-        word_order = Endian.Big
-        byte_order = Endian.Big
-
-        if order_string is None:
-            return (byte_order, word_order)
```

Comment:
it was not actually possible to get here as the class can't be invoked with a None value for `order_string`, so I changed the default case to handle an unrecognised/empty string instead of a None value